doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -74,7 +74,8 @@ See `Samples`_ for lists of changes for the protocol-related samples.
 Bluetooth® LE
 -------------
 
-|no_changes_yet_note|
+* Added host extensions for Nordic-only vendor-specific command APIs.
+  Implementation and integration of the host APIs can be found in the :file:`host_extensions.h` header and :file:`host_extensions.c` source code file.
```

Comment:
Do users need to know which source file this is implemented in? Maybe just have a link to the generated docs for the header file?